[CN-1261]: Add docs for lite members (#205)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/lite-members.adoc

= Enabling Lite Members

> To be able to enable lite members Hazelcast Platform Operator accepts a parameter `liteMemberCount` in Hazelcast CR configuration.

Reviewer: ```suggestion
To enable lite members, set the `liteMemberCount` parameter in the Hazelcast CR configuration.
```

.Example Configuration
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/hazelcast-lite-members.yaml[]
----

> There are some limitations while setting `liteMemberCount`:

Reviewer: ```suggestion
Bear in mind the following limitations while setting `liteMemberCount`:
```
